app/helpers/houses_helper.rb

```diff
@@ -22,18 +19,6 @@ def int_to_gender(int)
   #   check_it(@house.checkbox.refrigerator)
   #   # => '&checkmark;'
   def check_it(value)
-    value = case value
-            when true
-              '&checkmark;'
-            else
-              '&cross;'
-    end
-
-    value.html_safe
-  end
-
-  # TODO: DRY it. HousesController has same method.
-  def owner?
-    true if @house.user == current_user
+    value == true ? '&checkmark;'.html_safe : '&cross;'.html_safe
```

Comment:
Value will be boolean (it comes from Active Record), so we don't need to use `== true`.

```suggestion
    value ? '&checkmark;'.html_safe : '&cross;'.html_safe
```